beta/src/components/MDX/Sandpack/CustomPreset.tsx

```diff
@@ -70,11 +73,14 @@ export function CustomPreset({
               extensions={lintExtensions}
               extensionsKeymap={emptyArray}
             />
-            <Preview
-              className="order-last xl:order-2"
-              isExpanded={isExpanded}
-              lintErrors={lintErrors}
-            />
+            <div className="sp-stack h-full">
```

Comment:
This breaks the narrow view.

<img width="970" alt="Screenshot 2022-06-10 at 20 00 32" src="https://user-images.githubusercontent.com/810438/173132719-9360e276-fb14-4df4-9998-187ad789cacf.png">